src/mlpack/core/util/program_doc.cpp

```diff
@@ -33,14 +33,14 @@ using namespace std;
  *     information; each pair is <description, url>.
  */
 ProgramDoc::ProgramDoc(
-    const std::string& programName,
-    const std::string& shortDocumentation,
-    const std::function<std::string()>& documentation,
-    const std::vector<std::pair<std::string, std::string>>& seeAlso) :
-    programName(programName),
-    shortDocumentation(shortDocumentation),
-    documentation(documentation),
-    seeAlso(seeAlso)
+    const std::string programName,
+    const std::string shortDocumentation,
+    const std::function<std::string()> documentation,
+    const std::vector<std::pair<std::string, std::string>> seeAlso) :
+    programName(std::move(programName)),
+    shortDocumentation(std::move(shortDocumentation)),
+    documentation(std::movie(documentation)),
```

Comment:
Hey @y2s82, Could you please  change `movie` to `move`. This is causing the build failure. Thanks for all the work! 👍

Reply:
done